helios-tools/src/main/java/com/spotify/helios/cli/command/RollingUpdateCommand.java

```diff
@@ -131,6 +132,15 @@ public Long get() {
         .setDefault(EMPTY_TOKEN)
         .help("Insecure access token meant to prevent accidental changes to your job "
               + "(e.g. undeploys).");
+
+    ignoreFailuresArg = parser.addArgument("--ignore-failures")
+        .setDefault(false)
+        .action(storeTrue())
+        .help("When specified, the rolling update will continue on to all hosts in the deployment "
+              + "group without waiting for the job to reach the RUNNING state on each host. "
+              + "Be *very* careful about using this option, as it has the potential to completely "
+              + "take down your service by rolling out a broken job to all of the hosts in your "
```

Comment:
Would it help the user to enumerate all the checks that are skipped that are usually checked for AWAIT_RUNNING? e.g. retrieving job status, job unexpectedly undeployed, etc.